src/ai-dialog-body.ts

```diff
@@ -1,4 +1,4 @@
-import {customElement, inlineView} from 'aurelia-templating';
+import { customElement, inlineView } from 'aurelia-templating';
```

Comment:
Is the linter set up to enforce spacing here after open and before close brackets?